sw/device/silicon_creator/mask_rom/mask_rom.c

```diff
@@ -94,6 +99,9 @@ static void mask_rom_init(void) {
       .sink = uart_sink,
   });
 
+  sec_mmio_check_values(/*rnd_offset=*/0);
```

Comment:
q: Do you plan to leave this as `0`? If not, maybe we should add a TODO or create an issue.

Reply:
Done. Created an issue.